src/Core/Domain/Product/QueryResult/CategoryInformation.php

```diff
@@ -37,21 +37,31 @@ class CategoryInformation
      */
     private $id;
 
+    /**
+     * @var string
+     */
+    private $displayName;
+
     /**
      * @var array<int, string>
+     * @todo: this doesn't need to be localized because we are not updating category info in product page
+     *        (product name, description are localized, because all languages are needed for update)
```

Comment:
You can refacto this one with a single string If the query doesn't handle a LanguegId and the expected result is only one language there is no reason to keep some mixed data (some localized some not) So it's better to clearly state the contract of this query
BC break is ok here